worker/src/routers/apiRouter.ts

```diff
@@ -338,43 +337,70 @@ export const getAPIRouter = (router: BaseRouter) => {
   );
 
   router.put(
-    '/request/:id/property',
+    "/v1/request/:id/property",
     async (
       { params: { id } },
       requestWrapper: RequestWrapper,
       env: Env,
       _: ExecutionContext
     ) => {
       const client = await createAPIClient(env, requestWrapper);
-      const authParams = await client.db.getAuthParams();
-      
-      interface Body {
-        key: string,
-        value: string,
-      }
-      if (authParams.error !== null) {
-        return client.response.unauthorized();
-      }      
-      const property = await requestWrapper.getJson<Body>();
-      const {data, error} = await client.db.getRequestById(id);
 
+      const { data, error } = await client.db.getRequestById(id, false);
       if (error) {
         return client.response.newError(error, 500);
       }
-      
-      if (!data) {
+
+      if (!data || !data.helicone_org_id) {
         return client.response.newError("Request not found.", 404);
       }
 
+      const orgId = data.helicone_org_id;
+      const authParams = await client.db.getAuthParams(orgId);
```

Comment:
I think the auth will ALWAYS have the org id attached. There is no reason it shouldn't, right? It feels weird that we grab the requestId first and then do the auth check